src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -346,7 +350,9 @@ export class MeetingManager implements AudioVideoObserver {
 
   selectAudioOutputDevice = async (deviceId: string): Promise<void> => {
     try {
-      await this.audioVideo?.chooseAudioOutputDevice(deviceId);
+      if(this.supportsSetSinkId()) {
+        await this.audioVideo?.chooseAudioOutputDevice(deviceId);
+      }
       this.selectedAudioOutputDevice = deviceId;
       this.publishSelectedAudioOutputDevice();
```

Comment:
Are you sure that this is the correct fix?

After all, you *did not* select the audio output device, so why assign `selectedAudioOutputDevice` and publish?

I suspect that you want one of two things:

* Don't change `selectAudioOutputDevice` at all. If the caller wants to call it, they need to check for `setSinkId` support. Make sure that nothing else in the component library calls `selectAudioOutputDevice`.
* Bail out at the start of the function instead.

Reply:
Fixed. Went with option #1